### PR TITLE
UIIN-1097 Move items between holdings

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Instance/ItemsList/ItemsList.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  useMemo,
+  useCallback,
+  useState,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import { Droppable } from 'react-beautiful-dnd';
+import {
+  FormattedMessage,
+  injectIntl,
```
Comment (AndreiShumski2) [inline src/Instance/ItemsList/ItemsList.js:11]: useIntl?

File: src/Instance/ItemsList/ItemsListRow.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      className={rowClass}
+      role="row"
+      tabIndex="0"
+
+      style={rowStyles}
+    >
+      {
+        snapshot.isDragging
+          ? (
+            <span>Dragging</span>
```
Comment (AndreiShumski2) [inline src/Instance/ItemsList/ItemsListRow.js:50]: should it be translated?

File: src/Item/CreateItem/CreateItem.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+import ItemForm from '../../edit/items/ItemForm';
+
+const CreateItem = ({
+  stripes,
+  referenceData,
+  mutator,
+  instanceId,
+  holdingId,
+}) => {
+  const history = useHistory();
```
Comment (aliaksei-chumakou) [inline src/Item/CreateItem/CreateItem.js:30]: will it work on testing, where we don't have react-router v5?

File: src/ViewInstance.js [hunk 4]
```diff
@@ -173,6 +184,39 @@ class ViewInstance extends React.Component {
...
+    this.setState((prevState) => ({ isItemsMovement: !prevState.isItemsMovement }));
+  };
+
+  moveItems = (fromHolding, toHolding, items) => {
+    // TODO: replace temporary solution with correct one when implemented
+    const { mutator } = this.props;
+
+    return mutator.movableItems.GET({
+      params: {
+        query: items.map(item => `id=${item}`).join(' or '),
```
Comment (aliaksei-chumakou) [inline src/ViewInstance.js:197]:
```suggestion
        query: items.map(item => `id==${item}`).join(' or '),
```

File: src/Instance/ItemsList/ItemsListRow.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/Instance/ItemsList/ItemsListRow.js:50]: yesp, request is WIP

File: src/Item/CreateItem/CreateItem.js (hunk 3 above)
Comment (NikitaSedyx) [reply src/Item/CreateItem/CreateItem.js:30]: good question, seems no

@zburke are there any plans to bump react-router version in stripes?

**Comment by NikitaSedyx** [comment]
@SergiyVSergiyenko @mkuklis @VictorSoroka1 could you please review this one?

File: src/Item/CreateItem/CreateItem.js (hunk 3 above)
Comment (zburke) [reply src/Item/CreateItem/CreateItem.js]: We've had plans to bump `react-router` to `v5` for ages now. [STRIPES-674](https://issues.folio.org/browse/STRIPES-674) captures the work. I started this back in September and got 90% of the way there, including PRs in all platform-core apps, before getting hung up on [STCOR-390](https://issues.folio.org/browse/STCOR-390) and needing to drop it and move on to other things. I still feel badly about that 😞

**Comment by NikitaSedyx** [comment]
thank @zburke , hopefully I'll have a chance to review this issue

**Pull request merged**